packages/ui/src/components/button-dropdown/components/heading/heading.component.tsx

```diff
@@ -12,3 +12,4 @@ export function Heading({ className, tag: Tag = 'h1', children, ...props }: Head
     </Tag>
   );
 }
+Heading.displayName = 'ButtonDropdown.Heading';
```

Comment:
@kenjishiromajp @tomjackking @jaortiz We may need to keep eye on this issue in the future. When the component name is the same, react-docgen-typescript package override props table which create invalid docs.